public/stylesheets/masters/dark_mode/dark_mode_site_screen_.css

```diff
@@ -0,0 +1,750 @@
+    box-shadow: none;
+}
+
+.mce-container input:focus {
+  background: #f3efec;
+}
+
+/* 08-ACTIONS */
+
+.actions a,
+.actions a:link,
+.action,
+.action:link,
+.actions button,
+.actions input,
+input[type="submit"],
+button,
+.current,
+.actions label {
+  background: #555;
+  border-color: #222;
+  color: #eee;
+    box-shadow: inset 0 -8px 4px #232323, inset 0 8px 7px #555;
+    text-shadow: none;
+}
+
+.actions a:visited,
+.action:visited,
+.action a:link,
+.action a:visited {
+  color: #eee;
+}
+
+.actions a:hover,
+.actions button:hover,
+.actions input:hover,
+.actions a:focus,
+.actions button:focus,
+.actions input:focus,
+label.action:hover,
+.action:hover,
+.action:focus {
+  border-color: #000;
+  color: #fafafa;
+    box-shadow: inset 2px 2px 2px #000;
+}
+
+.actions a:active,
+.current,
+a.current,
+a:link.current,
+.current a:visited,
+span.unread,
+.replied,
+span.claimed,
+.actions span.defaulted {
+  background: #555;
+  border-color: #fff;
+  color: #fff;
+    box-shadow: inset 1px 1px 3px #333;
+}
+
+.current,
+a.current,
+a:link.current {
+  background: #222;
+  border-color: #555;
+    box-shadow: -1px -1px 3px #000;
+}
+
+.actions label.disabled {
+  background: #222;
+}
+
+.actions .disabled select {
+  color: #999;
+  border-color: #000;
+}
+
+.delete a,
+span.delete {
+  color: #ffada8;
+}
+
+.secondary {
+  background: #2a2a2a;
+  border-color: #555;
+    box-shadow: 1px 1px 3px #000;
+}
+
+/* don't let iOS add its special styling for dark buttons */
+button,
+input[type="button"],
+input[type="submit"] {
+    -webkit-appearance: none;
+}
+
+/* 09-ROLES-STATES */
+
+.own,
+.draft,
+.draft .wrapper,
+.unread,
+.child,
+.unwrangled,
+.unreviewed {
+  background: #222;
+  border-color: #555;
+}
+
+span.unread,
+.replied,
+span.claimed,
+.actions span.defaulted {
+  background: #222;
+  border-color: #555;
+    box-shadow: -1px -1px 3px #000;
+}
+
+.draggable,
+.droppable,
+span.requested,
+.nominations .rejected {
+  color: #ffada8;
+}
+
+.actions span.defaulted {
+  color: #eee;
+}
+
+span.offered,
+.replied,
+.nominations .approved {
+  color: #a8ffad;
+}
+
+.nominations .approved,
+.nominations .rejected {
+  background: #333;
+}
+
+span.offered.requested {
+  color: #ffe6a8;
+}
+
+/* 10-TYPES-GROUPS */
+
+a.tag:hover,
+.listbox .heading a.tag:visited:hover {
+  background: #95c6f4;
+  color: #111;
+}
+
+.wrapper {
+    box-shadow: 1px 1px 3px #000;
+}
+
+dl.index dd,
+li.relationships a {
+  background: #2a2a2a;
+}
+
+a.work {
+  color: #95c6f4;
+}
+
+.actions .reindex a {
+  border-bottom-color: #95c6f4;
+}
+
+span.series .divider {
+  color: #999;
+}
+
+.statistics .index li:nth-of-type(even) {
+  background: #333;
+}
+
+#inner .module .heading {
+  border-color: #555;
+}
+
+/* 11-GROUP-LISTBOX */
+
+.listbox,
+fieldset fieldset.listbox {
+  background: #2a2a2a;
+  border-color: #555;
+    box-shadow: 1px 1px 3px #000;
+}
+
+/* this should have a direct child selector but cannot, so be careful */
```

Comment:
This comment is an amazing testament to how long I have been working on this. I'll take it out and add the direct child selector. This was fixed in PR #2015 almost ten years ago.